src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -686,6 +686,7 @@ Microsoft.CodeAnalysis.Semantics.UnaryOperationKind.UnsignedPrefixIncrement = 77
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveGenericsOptions(Microsoft.CodeAnalysis.SymbolDisplayGenericsOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveLocalOptions(Microsoft.CodeAnalysis.SymbolDisplayLocalOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveMiscellaneousOptions(Microsoft.CodeAnalysis.SymbolDisplayMiscellaneousOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
+Microsoft.CodeAnalysis.SymbolDisplayFormat.WithTypeQualificationStyle(Microsoft.CodeAnalysis.SymbolDisplayTypeQualificationStyle typeQualificationStyle) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
```

Comment:
If this is just a missing 'Wither', then we should ahve this.  It looks like we've missed a unch of standard operations on SymbolDisplayFormat.